packages/ua-blocker/examples/demo.ts

```diff
@@ -1,6 +1,6 @@
-import { uaBlocker } from '@hono/ua-blocker'
-import { nonRespectingAiBots, useAiRobotsTxt } from '@hono/ua-blocker/ai-bots'
```

Comment:
Referencing `@hono/ua-blocker` from within the `src` directory is causing test failures in both #1178 and #1220. I've moved `demo.ts` out of `src` into a new `examples` directory. There might be other ways to do this, but it should unblock others for now..